### PR TITLE
Add script 1 for broad annotation

### LINKED ISSUE #110: Proposed Analysis: Add script 1 for broad annotation
## What are the scientific goals of the analysis?

To add script 1 for broad annotation, i.e., `01-cell-types-annotation-SingleR-broad.Rmd`.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

To add script 1 for broad annotation, i.e., `01-cell-types-annotation-SingleR-broad.Rmd`.

## What GitHub issue does your pull request address?

Closes #110.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review for documentation and code logic and clarity.

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+  mutate(cells_number_all = Freq) %>% 
+  dplyr::select(-Freq) %>%
+  left_join(summary_table_pruned)
+```
+
+We will check and compare the pruned labels and the initial labels by summarizing the identified broad cell type annotations.
+
+```{r, fig.align = "left", results = "asis", message = FALSE, warning = FALSE, echo = FALSE}
+tables1 <- summary_table
+cat("  \n<div align=\"center\", style=\"font-size:80%\">  \n")
```
Comment (sharonfreshour) [inline analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd:240]: I think this should be updated to match `pandoc` bug fix from PR #115 

```suggestion
cat("  \n<div align=\"center\" style=\"font-size:80%\">  \n")
```

File: analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+summary_table <- table(cell_type_broad = seurat_obj@meta.data$labels, Removed = to.remove) %>%
+  as.data.frame() %>% 
+  pivot_wider(names_from = "Removed", values_from = "Freq") 
+```
+
+We will summarize the identified broad cell type annotations to be removed.
+
+```{r, fig.align = "left", results = "asis", message = FALSE, warning = FALSE, echo = FALSE}
+tables1 <- summary_table
+cat("  \n<div align=\"center\", style=\"font-size:80%\">  \n")
```
Comment (sharonfreshour) [inline analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd:285]: I think this should be updated to match `pandoc` bug fix from PR #115 

```suggestion
cat("  \n<div align=\"center\" style=\"font-size:80%\">  \n")
```

File: analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+print(knitr::kable(tables1, caption = "Broad cell type annotations to be removed"))
+cat("  \n</div>  \n")
+cat("  \n\\pagebreak  \n")
+```
+
+Low deltas indicate that the assignment is uncertain, which is especially relevant if the cell’s true label does not exist in the reference. This entire process can be visualized ans inspected using the plotScoreDistribution() function, which displays the per-label distribution of the deltas across cells.
```
Comment (sharonfreshour) [inline analyses/cell-types-annotation/01-cell-types-annotation-SingleR-broad.Rmd:291]:
```suggestion
Low deltas indicate that the assignment is uncertain, which is especially relevant if the cell’s true label does not exist in the reference. This entire process can be visualized and inspected using the plotScoreDistribution() function, which displays the per-label distribution of the deltas across cells.
```

**Review by sharonfreshour — APPROVED** [review]
Looks good, code is clear and logical. 

I made a few suggestions related to table styles and the `pandoc` bug brought up in PR #115 and corrected one typo in the report text.

**Comment by AntoniaChroni** [comment]
> Looks good, code is clear and logical.
> 
> I made a few suggestions related to table styles and the `pandoc` bug brought up in PR #115 and corrected one typo in the report text.

Thank you! All these were addressed here 1e5dd02fdd26bda73a67f9313a26056b7e963d1c.

**Pull request merged**